R/appenders.R

```diff
@@ -232,6 +232,34 @@ appender_pushbullet <- function(...) {
 }
 
 
+#' Send log messages to ntfy
+#' @param ... parameters passed to [ntfy::ntfy_send], such as `title`
+#'     or `tags`. 
+#'     
+#' @details Configure server and topic via environment variables. See 
+#' [ntfy::ntfy_topic()] for details.     
+#'     
+#' @export
+#' @note This functionality depends on the \pkg{ntfy} package.
+#' @family log_appenders
+#' @export
+appender_ntfy <- function(title = "{logger}", 
+                          tags = c("memo"),
```

Comment:
Sorry, I'm not sure how good this default value might be -- is this something specific to `ntfy.sh`? I mean why `"memo"`, also why the `c` wrapper?

Reply:
The `tags` sets the emoji (see screenshot) and I used an explicit vector to demonstrate the expected structure (not a list). Both of these can be changed if preferred.